views/facethistory.hbs

```diff
@@ -81,19 +77,13 @@
 			<br /><br />
 			<a id="queryBuilderLink" target="blank" href="">/facetHistory/</a>
 		</form>
-
-		<br /><br />
-		<br />
-
+		<br /><br /> <br />
```

Comment:
Might be worth adding padding/margin insted of line breaks.

Reply:
Fixed 👍